### PR TITLE
Using gradle

### PR DESCRIPTION
**Pull request by atul1805**
Removed maven and using gradle instead. Also updated dependencies.

For setup, run `./gradlew idea` and open `etarde.ipr` file in intellij.

### CODE REVIEW

**Comment by arjun4084346** [comment]
thank you for the changes.
log4j version is still in the vulnerable range [1.2, 1.2.27]

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+.DS_Store
+
+# vi
+*.swp
+
+# Build files
+target/
+
+# plugin generated
+lombok.config
```
Comment (arjun4084346) [inline .gitignore]: Please add these files also.
*.log
.idea
target
src/main/resources/oauth.properties

**Comment by atul1805** [comment]
> thank you for the changes.
> log4j version is still in the vulnerable range [1.2, 1.2.27]

Lates is `1.2.17` https://mvnrepository.com/artifact/log4j/log4j

File: .gitignore (hunk 1 above)
Comment (atul1805) [reply .gitignore]: these files are already in gitignore. Don't see `oauth.properties` this file though.

**Pull request merged**